File: lv0/triangle_complete/best.php
<?php

function solution($sides) {
    $longest = max($sides);
    unset($sides[array_search($longest, $sides)]);
    return $longest < array_sum($sides) ? 1 : 2;
}

$params = [
    [[1, 2, 3], 2],
    [[3, 6, 2], 2],
    [[199, 72, 222], 1],
];

foreach($params as $param) {
    $expected = $param[1];
    $result = solution($param[0]);
    echo "Expected: {$expected}, Result: {$result}\n";
}

?>

File: lv0/taller/best.py
def solution(array, height):
    count = 0
    for h in array:
        if h > height:
            count += 1
    return count

params = [
    ([149, 180, 192, 170], 167, 3),
    ([180, 120, 140], 190, 0),
]

for param in params:
    result = solution(param[0], param[1])
    print(result)